src/zcbor_encode.c

```diff
@@ -356,13 +358,13 @@ bool zcbor_map_end_encode(zcbor_state_t *state, uint32_t max_num)
 bool zcbor_nil_put(zcbor_state_t *state, const void *input)
 {
 	(void)input;
-	return primx_encode(state, 22);
+	return primitive_put(state, 22);
```

Comment:
Should you change the `input` parameter name to `unused` as you did in the header file?